quisp/test_utils/StaticEnv.h

```diff
@@ -0,0 +1,151 @@
+#pragma once
+
+#include <omnetpp.h>
+#include "Configuration.h"
+
+namespace quisp_test {
+namespace env {
+
+using namespace omnetpp;
+
+/**
+ * \brief cEnvir implementation for unit testing based on OMNeT++'s StaticEnv.
+ * \see https://github.com/omnetpp/omnetpp/blob/308b3b2b4c060b165911237a9bd9470ea010d34c/src/sim/csimulation.cc#L728
+ */
+class StaticEnv : public omnetpp::cEnvir {
```

Comment:
This class supports some environmental functions of omnet, right?

Reply:
yes, it inherits cEnvir. 
and I added some utility methods for testing